onnxruntime/core/providers/cpu/math/gemm.cc

```diff
@@ -249,6 +265,48 @@
   return Status::OK();
 }
 
+template <>
+Status Gemm<MLFloat16>::Compute(OpKernelContext* context) const {
+  concurrency::ThreadPool* thread_pool = context->GetOperatorThreadPool();
+
+  const auto* A = context->Input<Tensor>(0);
+  const auto* B = packed_b_ ? nullptr : context->Input<Tensor>(1);
```

Comment:
>packed_b_ ? nullptr : context->Input<Tensor>(1); [](http://example.com/codeflow?start=18&length=48)

nit: no need to check packed_b_

Reply:
Would you mind elaborating more? So, only the float32 case may have a packed_b_ ?